client/src/components/UserMakeInstrument.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
```

Comment:
For this PR, the `UserMakeInstrument` component is out of scope. Please make a separate PR for this feature